internal/tasks/device_render.go

```diff
@@ -605,3 +607,61 @@ func (t *DeviceRenderLogic) cloneCachedGitRepoToIgnition(ctx context.Context, re
 	ignToReturn := wrapper.AsIgnitionConfig()
 	return &ignToReturn, nil
 }
+
+// ignitionConfigToRenderedConfig converts an ignition config to rendered config bytes
+func ignitionConfigToRenderedConfig(ignition *config_latest_types.Config) ([]byte, error) {
```

Comment:
Why convert to ignition, and from ignition to inline, rather than converting directly to inline?

Reply:
it was only to avoid implementing a merge. The goal is to remove.

Reply:
https://issues.redhat.com/browse/EDM-1167